tuxedo-template-runtime/src/lib.rs

```diff
@@ -170,6 +171,8 @@ pub enum OuterConstraintChecker {
     Money(money::MoneyConstraintChecker<0>),
     /// Checks Free Kitty transactions
     FreeKittyConstraintChecker(kitties::FreeKittyConstraintChecker),
+    /// Checks tradable Kitty transactions
+    TradableKittyConstraintChecker(tradable_kitties::TradableKittyConstraintChecker<0>),
```

Comment:
Small grumble: The variant name should not contain "constraint checker". They are all constraint checkers. I realize this was already the case for the free kitty one, and you may fix it there too if you like.

Reply:
Thanks. I will update both below:
    FreeKitty(kitties::FreeKittyConstraintChecker)
    TradableKitty(tradable_kitties::TradableKittyConstraintChecker<0>)